Client/Connection.ts

```diff
@@ -29,6 +29,12 @@ export class Connection {
 				...header,
 				"Content-Type": "application/json; charset=utf-8",
 			}
+		if (path.includes("auth/passwordreset") || path.includes("auth/login"))
+			headers = {
+				...header,
+				"Content-Type": "application/json; charset=utf-8",
+				"x-invoking-system": "portal_2",
+			}
```

Comment:
you can just add the x-invoking-system on every call, thats nicer than checking paths

you shouldnt need to be setting the content type explicitly here


(the reason it wasnt working before was because you have header and header_S_, single character difference names are prone to this sort of problem)